src/Writing/Writer.php

```diff
@@ -86,8 +87,8 @@ protected function writePostmanCollection(array $groups): void
                 $collectionPath = "{$this->staticTypeOutputPath}/collection.json";
                 file_put_contents($collectionPath, $collection);
             } else {
-                Storage::disk('local')->put("{$this->docsName}/collection.json", $collection);
-                $collectionPath = Storage::disk('local')->path("$this->docsName/collection.json");
+                Storage::disk('local')->put($this->docsName->getScribeConfigFile() . "/collection.json", $collection);
+                $collectionPath = Storage::disk('local')->path($this->docsName->getScribeConfigFile() . "/collection.json");
```

Comment:
If we're doing this, perhaps we should move away from concatenation, and just make it a resolver (a la Laravel's `storage_path(...)`)? Somethig like:

```php
Storage::disk('local')->path($this->paths->outputFile("/collection.json"));
```